sigstore/_cli.py

```diff
@@ -34,10 +35,24 @@ def main():
     type=click.File("rb"),
     default=resources.open_binary("sigstore._store", "ctfe.pub"),
 )
+@click.option(
+    "oidc_client_id", "--oidc-client-id", type=click.STRING, default="sigstore"
+)
+@click.option(
+    "oidc_client_secret", "--oidc-client-secret", type=click.STRING, default=str()
+)
+@click.option(
+    "oidc_issuer",
+    "--oidc-issuer",
+    type=click.STRING,
+    default="https://oauth2.sigstore.dev/auth",
```

Comment:
cc @di thoughts on removing the `oidc` prefix from these flags? AFAICT they don't have equivalents in `cosign`, so changing them here wouldn't be a compatibility concern and would make the CLI slightly less verbose.

Reply:
These flags come from `cosign sign`. They're experimental though which is why you may not have seen them. From the help text:
```
      --oidc-client-id string                                                                    [EXPERIMENTAL] OIDC client ID for application (default "sigstore")
      --oidc-client-secret string                                                                [EXPERIMENTAL] OIDC client secret for application
      --oidc-issuer string                                                                       [EXPERIMENTAL] OIDC provider to be used to issue ID token (default "https://oauth2.sigstore.dev/auth")
```

Reply:
Thanks! Okay, let's keep them as-is then.

Reply:
+1, we should try to stay as close as possible to `cosign`'s CLI.